parse/parse.go

```diff
@@ -39,9 +39,9 @@ func ParseSha1File(path string) (sha1s []types.Sha1SBOM, err error) {
 }
 
 func parseSpaceSeperatedLocationAndSha1(scanner *bufio.Scanner) (sha1 types.Sha1SBOM) {
-	s := strings.Split(scanner.Text(), " ")
-	sha1.Location = s[0]
-	sha1.Sha1 = s[1]
+	s := strings.Split(scanner.Text(), "  ")
```

Comment:
Very cool how this is now validated by a test. 

Do we need to document the double-space delimiter, maybe in the help text of the `fry` command? And/or in the README.md?

Reply:
Yeah, I can do that. Good point!